ports/espressif/boards/adafruit_feather_esp32s3_4mbflash_2mbpsram/mpconfigboard.h

```diff
@@ -45,3 +45,5 @@
 #define DEFAULT_UART_BUS_TX (&pin_GPIO39)
 
 #define DOUBLE_TAP_PIN (&pin_GPIO34)
+
+#define CIRCUITPY_SETTABLE_PYSTACK 0
```

Comment:
If this is a chip level change, it can go in `mpconfigport.h`
```c
// Temporarily disable settable pystack
#if defined(CONFIG_IDF_TARGET_ESP32C3) || defined(CONFIG_IDF_TARGET_ESP32S3)
#undef CIRCUITPY_SETTABLE_PYSTACK
#define CIRCUITPY_SETTABLE_PYSTACK 0
#endif
```

Reply:
Yea this is a hundred times better.